Pages/Index.cshtml.cs

```diff
@@ -55,7 +55,7 @@ public void OnGet(string jsonSting)
                 QuickTypeCrimes.Crime[] crimes = QuickTypeCrimes.Crime.FromJson(crimeJSON);
                 
                
-                string propertyResJason = webClient.DownloadString("https://data.cincinnati-oh.gov/resource/m76i-p5p9.json?project_type=RESIDENTIAL");
+                string ResidentialPropertyType = webClient.DownloadString("https://data.cincinnati-oh.gov/resource/m76i-p5p9.json?project_type=RESIDENTIAL");
```

Comment:
Renamed the variable which describes Residential Property Types. Reduces technical debt by improving the readability quotient of the code. Also, the necessity  of inline comments is reduced.

Sources
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions

Reply:
Good.  More readable.

We often name variables lowerCamelCase (though C# doesn't always use this convention).